scripts/deploy

```diff
@@ -96,6 +99,20 @@ else
   fi
 fi
 
+# check for `npm`
+if ! which npm; then
+  echo "npm needs to be installed to use this script"
+  exit 1;
+fi
+if [ $(npm --version | cut -f1 -d.) -lt 9 ]; then
+  echo "npm needs to be version 9 or above to use this script"
+  exit 1;
+fi
+if [ $(nodejs --version | tr -d v | cut -f1 -d.) -lt 18 ]; then
```

Comment:
I don't know the 'usual' way on installing node/npm, but with `nvm` the node version is what is selected. So it's hard to tell from the "npm needs to be version 9 or above to use this script" message which node version I should install. Because of this, I would suggest putting the node check before the npm check.

Reply:
swapped~